server/tilesource/tiff.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, item, **kwargs):
 
         largeImagePath = self._getLargeImagePath()
         lastException = None
+        self._associatedImages = {}
```

Comment:
It isn't obvious to me the distinction between associated and embedded images.  Could you add a comment describing how they are different?

Reply:
I'll add a comment.  Associated images are images stored the "usual" way in a TIFF file that aren't tiled.  Embedded images are vendor-specific extensions where an image is put into a normally non-image tag and stored as metadata on a usual TIFF image.